cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -212,36 +189,6 @@ def test_staff_course_listing(self, default_store, mongo_calls):
         with check_mongo_calls(mongo_calls):
             list(_accessible_courses_summary_iter(self.request))
 
-    @ddt.data(
-        (ModuleStoreEnum.Type.split, 'xmodule.modulestore.split_mongo.split_mongo_kvs.SplitMongoKVS'),
-        (ModuleStoreEnum.Type.mongo, 'xmodule.modulestore.mongo.base.MongoKeyValueStore')
-    )
-    @ddt.unpack
-    def test_errored_course_regular_access(self, store, path_to_patch):
```

Comment:
+ Also this?

Reply:
Same reason as described above. This is just another test for different type of users.